Geometry/HGCalCommonData/interface/HGCalCellOffset.h

```diff
@@ -0,0 +1,25 @@
+#ifndef Geometry_HGCalCommonData_HGCalCellOffset_h
+#define Geometry_HGCalCommonData_HGCalCellOffset_h
+
+#include <cmath>
+#include <cstdint>
+#include <vector>
+#include "Geometry/HGCalCommonData/interface/HGCalCell.h"
+
+class HGCalCellOffset {
+public:
+  HGCalCellOffset(double waferSize, int32_t nFine, int32_t nCoarse, double guardRingOffset_, double mouseBiteCut_);
+
+  std::pair<double, double> cellOffsetUV2XY1(int32_t u, int32_t v, int32_t placementIndex, int32_t type);
+  double cellAreaUV(int32_t u, int32_t v, int32_t placementIndex, int32_t type);
+
+private:
+  const double sqrt3_ = std::sqrt(3.0);
+  const double sqrt3By2_ = (0.5 * sqrt3_);
+  std::vector<std::vector<std::vector<double>>> offsetX, offsetY;
```

Comment:
Is there any measure of the memory needed to host these `vector-vector-vector` s?

Reply:
They have dimension (2,3,6) giving 288 bytes and 24 bytes for container

Reply:
If the sizes are known at compile time, would it make sense to use some fixed-size container, instead?

Reply:
Changed to a fixed size array container